.gitignore

```diff
@@ -36,6 +36,7 @@ captures/
 *.iml
 .idea/*
 !.idea/codeStyleSettings.xml
+.idea/
```

Comment:
You can use `git update-index --skip-worktree .idea/$FILENAME` to ignore git-controlled files with keeping changes. 😃